src/mtev_rest.c

```diff
@@ -447,6 +454,8 @@ mtev_http_rest_new_rule_auth_closure(const char *method, const char *base,
   }
   rule = calloc(1, sizeof(*rule));
   rule->method = strdup(method);
+  rule->nice_name = malloc(strlen(base) + strlen(expr) + 2);
```

Comment:
We don't have a way to remove (or free) rules that have been added... it will need to be address with the freeing of everything related to that rule if/when we add that capability.